enhancements/20221228-various-storage-class-support.md

```diff
@@ -0,0 +1,114 @@
+# Various StorageClass support of Harvester CSI Driver
+
+## Summary
+
+Harvester supports the user-defined StorageClass after v1.1.0. Because of that, Harvester CSI Driver should also let users use the customer StorageClass mentioned above.
```

Comment:
```suggestion
Harvester supports the user-defined StorageClass after v1.1.0. Because of that, Harvester CSI Driver should also let users use the custom StorageClass mentioned above.
```